.github/CONTRIBUTING.md

```diff
@@ -46,6 +46,9 @@ $ npm run <command>
 # run a compile of packages
 $ npm run compile
 
+# start watch mode (in a separate terminal)
+$ npm run watch
```

Comment:
How does this compare with the other `npm run watch` command (immediately after this one in the file) ?

Reply:
Mea culpa. Removing this.